docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -44,28 +44,29 @@ __Shorthand__: `-k`
 ### `--config-file`
 <p><small>| REQUIRED | STRING |</small></p>
 
-:::caution For YAML Configuration Files ONLY
-This is required only if you're using a YAML file to configure your tunnels.
-:::
-
 __Description__: Defines the local path to a YAML file containing a Sauce Connect Proxy configuration. For instructions, [Configuring Tunnels with a YAML File](/secure-connections/sauce-connect/setup-configuration/yaml-config/).<br/>
 __Enviroment variable__: n/a<br/>
 __Shorthand__: `-c`
 
+:::caution
+This is required only if you're using a YAML file to configure your tunnels.
+:::
 
 ---
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
-__Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Description__: Sets your Sauce Labs [data center region](#data-center-endpoints) (e.g., `us-west`, `eu-central`, `apac-southeast`).<br/>
+__Default__: If you don't specify a Data Center, Sauce Connect will default to `us-west`. <br/>
 __Enviroment variable__: `SAUCE_REGION`<br/>
 __Shorthand__: `-r`
```

Comment:
We could add code examples with the required options for each region, like you did for the --rest-url flag below. For example: `./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -r us-west`, etc.